src/data/zwave_js.ts

```diff
@@ -350,6 +369,19 @@ export const provisionZwaveSmartStartNode = (
     planned_provisioning_entry,
   });
 
+export const unprovisionZwaveSmartStartNode = (
+  hass: HomeAssistant,
+  entry_id: string,
+  dsk?: string,
+  node_id?: number
```

Comment:
Why is node ID optional? What happens if you leave it out?

Reply:
(Not sure if the device stays in the provisioning list after it is added to HA?!)